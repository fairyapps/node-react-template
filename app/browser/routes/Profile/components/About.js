import React, { Component } from "react";

const About = props => <h1>About Here</h1>;

export default About;
