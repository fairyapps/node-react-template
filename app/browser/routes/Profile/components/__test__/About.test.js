import React from "react";
import renderer from "react-test-renderer";
import About from "../About";

test("Aboutw renders correct snapshot", () => {
  const component = renderer.create(<About />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
