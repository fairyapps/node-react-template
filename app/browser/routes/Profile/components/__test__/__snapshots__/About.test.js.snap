// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aboutw renders correct snapshot 1`] = `
<h1>
  About Here
</h1>
`;
